helmfile/charts/grafana-dashboards/dashboards/cluster-api-autoscaling-dashboard.json

```diff
@@ -0,0 +1,1385 @@
+                      },
+                      "2": {
+                        "index": 2,
+                        "text": "NotAutoscaled"
+                      },
+                      "3": {
+                        "index": 3,
+                        "text": "NotUnneededLongEnough"
+                      },
+                      "4": {
+                        "index": 4,
+                        "text": "NotUnreadyLongEnough"
+                      },
+                      "5": {
+                        "index": 5,
+                        "text": "NodeGroupMinSizeReached"
+                      },
+                      "6": {
+                        "index": 6,
+                        "text": "MinimalResourceLimitExceeded"
+                      },
+                      "7": {
+                        "index": 7,
+                        "text": "CurrentlyBeingDeleted"
+                      },
+                      "8": {
+                        "index": 8,
+                        "text": "NotUnderutilized"
+                      },
+                      "9": {
+                        "index": 9,
+                        "text": "NotUnneededOtherReason"
+                      },
+                      "10": {
+                        "index": 10,
+                        "text": "RecentlyUnremovable"
+                      },
+                      "11": {
+                        "index": 11,
+                        "text": "NoPlaceToMovePods"
+                      },
+                      "12": {
+                        "index": 12,
+                        "text": "BlockedByPod"
+                      },
+                      "13": {
+                        "index": 13,
+                        "text": "UnexpectedError"
+                      }
+                    },
+                    "type": "value"
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 9,
+        "w": 24,
+        "x": 0,
+        "y": 19
+      },
+      "id": 27,
+      "options": {
+        "cellHeight": "sm",
+        "footer": {
+          "countRows": false,
+          "enablePagination": false,
+          "fields": "",
+          "reducer": [
+            "sum"
+          ],
+          "show": false
+        },
+        "showHeader": true,
+        "sortBy": []
+      },
+      "pluginVersion": "9.5.5",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PE090269EBE049DA8"
+          },
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "cluster_autoscaler_unremovable_nodes_count{pod=~\"^$cluster.*\"}",
+          "format": "table",
+          "instant": true,
+          "legendFormat": "__auto",
+          "range": false,
+          "refId": "A"
+        }
+      ],
+      "title": "Unremovable nodes",
```

Comment:
What does `__name__` show in this table? Is it the name of a node? In the screenshot I can just see the start of the values.
Is `receive` and `service` needed? If not, then I think you can remove those to give more space to the other columns.

Reply:
I removed `__name__`, `recieve`, and `service` as all were kinda irrelevant.